.github/workflows/ci.yml

```diff
@@ -0,0 +1,81 @@
+name: CI
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
+        rust: [ stable ]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Install toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: ${{ matrix.rust }}
+          override: true
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Build code
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --all
+
+      - name: Test code
+        uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: --all --all-features --all-targets
```

Comment:
This will fail because the cryptolib features are mutually exclusive.

Reply:
Oops, sorry. I thought I already resolved it in a prior commit. I removed now the remaining `--all-features` flags on my local branch and will force-push later when we resolved the remaining issues.